### ISSUE TITLE
`message` `FormatBlock` is exported twice

### ISSUE THREAD

**Issue by kpodsiad** [outr/scribe#270]
Starting from 3.7.0 `message` can be imported both via `import scribe._` or `import scribe.format._`. Both imports are very useful and often used together which causes compilation error now.

Reproductuon with [scala-cli](https://scala-cli.virtuslab.org/)
```scala
//> using scala "2.13.8"
//> using lib "com.outr::scribe:3.7.0"

import scribe._
import scribe.format._

object Scribe extends App {
  val defaultFormat: Formatter = formatter"$date $levelPaddedRight $message"
}
```

yields  
`reference to message is ambiguous;
it is imported twice in the same scope by
import scribe.format._
and import scribe._bloop`

I didn't see this change listed in release notes so I suppose it's unintentional.

**Comment by darkfrog26** [comment]
For the code you mention above, I don't think you need anything from the `scribe` package anyway.

**Comment by kpodsiad** [comment]
> For the code you mention above, I don't think you need anything from the scribe package anyway.

Yeah I know, it was just a reproduction ;)

I'm not a fan of wildcard imports and I always end with explicit ones, but sometimes they are convenient, especially when prototyping. I just wanted to let you know about this, let's call it, "regression".

**Comment by darkfrog26** [comment]
This is resolved and will go out in the next release (3.8.0). Note, the fix changes `format.message` to `format.messages` as it better reflects what that format block does now.
